src/main/java/me/totalfreedom/totalfreedommod/command/Command_rainbowtag.java

```diff
@@ -0,0 +1,44 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import java.util.Iterator;
+import static me.totalfreedom.totalfreedommod.command.Command_tag.FORBIDDEN_WORDS;

```

Comment:
Do we have to import it as a static? I seem to remember it usually being a bad idea?